clang/lib/CodeGen/CGDecl.cpp

```diff
@@ -2523,6 +2575,15 @@ llvm::Function *CodeGenModule::getLLVMLifetimeEndFn() {
   return LifetimeEndFn;
 }
 
+/// Lazily declare the @llvm.fake.use intrinsic.
+llvm::Function *CodeGenModule::getLLVMFakeUseFn() {
+  if (FakeUseFn)
+    return FakeUseFn;
+  FakeUseFn =
+      llvm::Intrinsic::getDeclaration(&getModule(), llvm::Intrinsic::fake_use);
+  return FakeUseFn;
```

Comment:
```suggestion
  if (!FakeUseFn)
    FakeUseFn =
        llvm::Intrinsic::getDeclaration(&getModule(), llvm::Intrinsic::fake_use);
  return FakeUseFn;
```
This seems a bit simpler to read, but I'm not insisting.